backend/ecs_tasks/delete_files/utils.py

```diff
@@ -3,7 +3,7 @@
 
 
 def remove_none(d: dict):
-    return {k: v for k, v in d.items() if v is not None and v is not ""}
+    return {k: v for k, v in d.items() if v is not None and v != ""}
```

Comment:
This is slightly unrelated, fixes a warning that occurs when running unit tests, introduced when we migrated to Python 3.9 in #322 

```
backend/ecs_tasks/delete_files/utils.py:6
  /home/runner/work/amazon-s3-find-and-forget/amazon-s3-find-and-forget/backend/ecs_tasks/delete_files/utils.py:6: SyntaxWarning: "is not" with a literal. Did you mean "!="?
    return {k: v for k, v in d.items() if v is not None and v is not ""}
-- Docs: https://docs.pytest.org/en/stable/warnings.html
```

Reply:
I had a wanring on this when I did the upgrade to 3.9, I should have changed it then, sorry.